--- Makefile	2022-10-29 01:00:00.000000000 +0000
+++ Makefile	2022-10-29 01:00:00.000000000 +0000
@@ -2,8 +2,8 @@
 LNCR_EXE=Manjaro.exe

 DLR=curl
-DLR_FLAGS=-L
-LNCR_ZIP_URL=https://github.com/yuk7/wsldl/releases/download/20121600/icons.zip
+DLR_FLAGS=--silent --location
+LNCR_ZIP_URL!=curl --silent https://api.github.com/repos/yuk7/wsldl/releases | jq --raw-output ".[0].assets[].browser_download_url" | grep --extended-regexp "icons.zip"
 LNCR_ZIP_EXE=Manjaro.exe

 all: $(OUT_ZIP)
@@ -11,7 +11,7 @@
 zip: $(OUT_ZIP)
 $(OUT_ZIP): ziproot
	@echo -e '\e[1;31mBuilding $(OUT_ZIP)\e[m'
-	cd ziproot; zip ../$(OUT_ZIP) *
+	cd ziproot; bsdtar -a -cf ../$(OUT_ZIP) *

 ziproot: Launcher.exe rootfs.tar.gz
	@echo -e '\e[1;31mBuilding ziproot...\e[m'
